### ISSUE TITLE
Running the latest release on Linux fails with a glib error

### ISSUE THREAD

**Issue by skorokithakis** [stronnag/bbl2kml#10]
Trying to run the latest release on Ubuntu 20.04.1 fails with:

```
otx2kml: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by otx2kml)
```

**Comment by skorokithakis** [comment]
Would linking statically be less problematic, perhaps?

**Comment by stronnag** [comment]
It is staticallly linked; however you can't escape a dynamic glibc dependency. Building on Debian Stable has an much older glibc, so it should more anywhere.

**Comment by stronnag** [comment]
OK, I've replaced the linux archives in the 0.8.0 release with files built in Ubuntu 20.04. Can you test that `otx2kml` please

**Comment by skorokithakis** [comment]
Works great, thanks! OTX logs are a bit sparse, as you said, but eh. Would you happen to know how I can get Google Earth to interpolate between two successive points? I fly over the same spot a lot and it's just a cloud of dots currently.

**Comment by stronnag** [comment]
Not sure what you mean. If you're trying to follow the track, the easiest way is to use the time slider; move the right knob to the left, then use either the step right control (right end of the slider) or the animation (clock icon) to step through the scenario. There's animation speed setting via the spanner icon.
![animate](https://user-images.githubusercontent.com/158229/103693118-aeb64d80-4f90-11eb-84f9-91308d5b3301.png)

**Comment by skorokithakis** [comment]
Ah, I thought I saw a screenshot where there was a "wall" under the points that connected them, maybe that wasn't Google Earth. Thank you!

**Comment by stronnag** [comment]
Wilko has used another tool to do that. You can build a curtain (walls are so 2020) using GE styling.
* Select the Flight Modes folder
* Properties =>  Style, Colour => Share Style
* Altitude => Extend to ground, OK

Unfortunately, you  then override the other styling and lose the Flight Mode colour coding. I guess I could add an "Extend" option ...

**Comment by wilco1967** [comment]
That 'wall' (or curtain... whatever.... ;-) was in google earth. 
Is was generated from the OTx logfile, opened from OpenTx companion log viewer, which in turn opens Google earth (you need to have the google earth executable defined in OTx companion settings to do that). Once shown In earth, under properties of the logfile, there is an option to 'extend to ground level', which creates the wall.

**Comment by stronnag** [comment]
Or you can try this with `extrude` and see if that works for you. (Linux 64bit binaries)

![Screenshot-20210105203327-133x179](https://user-images.githubusercontent.com/158229/103696380-25a21500-4f96-11eb-9649-baf4ea59002d.png)



[bbl2kml-0.8.1-x1.tar.gz](https://github.com/stronnag/bbl2kml/files/5772476/bbl2kml-0.8.1-x1.tar.gz)

**Comment by skorokithakis** [comment]
I'll try those, thank you! This issue can be closed, BTW.

**Comment by stronnag** [comment]
I even remembered to build on Ubuntu!

**Comment by stronnag** [comment]
Updated with better styling and proper style merging, so you can have f/s icon and extrude lines.

Not sure if I don't like this **better** than without the lines, OTOH, can always use `BBL2KML_OPTS=--extrude` 

[bbl2kml-0.8.1-x2-win32.zip](https://github.com/stronnag/bbl2kml/files/5772639/bbl2kml-0.8.1-x2-win32.zip)
[bbl2kml-0.8.1-x2-macos.zip](https://github.com/stronnag/bbl2kml/files/5772640/bbl2kml-0.8.1-x2-macos.zip)
[bbl2kml-0.8.1-x2-linux-x86_64.tar.gz](https://github.com/stronnag/bbl2kml/files/5772642/bbl2kml-0.8.1-x2-linux-x86_64.tar.gz)
[bbl2kml-0.8.1-x2-linux-ia32.tar.gz](https://github.com/stronnag/bbl2kml/files/5772644/bbl2kml-0.8.1-x2-linux-ia32.tar.gz)

Damn, I now know more about KML than I ever thought I'd need to.

**Comment by skorokithakis** [comment]
This is happening again with 0.9.6:

```
19:29:56 $ flightlog2kml --help
flightlog2kml: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by flightlog2kml)
```

**Comment by stronnag** [comment]
Damn it, must have missed the completely static flag .... let me try again

**Comment by stronnag** [comment]
The build script (and uploaded build) should be fixed now ...

**Comment by skorokithakis** [comment]
That worked, thanks for the quick response!
